.github/workflows/release.yml

```diff
@@ -22,7 +22,35 @@ jobs:
           npm install
           npm run build
 
+      - name: Get release version
+        id: get_version
+        run: echo ::set-env name=RELEASE_VERSION::$(echo ${GITHUB_REF:11})
+
+      - name: Publish check
+        run: |
+          ver=$RELEASE_VERSION
+          check_pkg() {
+            curl -sL "http://registry.npmjs.com/@pipcook/$1" | jq -r '.versions."'$2'".version'
+          }
+
+          check_image() {
+            curl -sL "https://hub.docker.com/v2/repositories/pipcook/pipcook/tags/$1" | jq -r '.name'
+          }
+
+          if
+            [ $(check_pkg pipcook-cli $ver) == $ver ] &&
+            [ $(check_pkg daemon $ver) == $ver ] &&
+            [ $(check_pkg pipboard $ver) == $ver ]
```

Comment:
We might not need to check version for pipboard, because we have moved that to a separate repository: https://github.com/imgcook/pipboard.

Reply:
Looks like the release of pipboard has separated with this repo, But if we add version-control for pipcook init in Dockerfile, it will install a specified version for pipboard and maybe failed without verification.

Reply:
I started a discussion at https://github.com/alibaba/pipcook/pull/481/files#r474507514, let's move there :)

Reply:
As the #481 has been merged which has removed a specific version for the board, IMO we are able to remove board check.

Reply:
Removed.